# Translations template for swift.
# Copyright (C) 2015 ORGANIZATION
# This file is distributed under the same license as the swift project.
#
# Translators:
# Andreas Jaeger <jaegerandi@gmail.com>, 2016. #zanata
# Pablo Caruana <pcaruana@redhat.com>, 2016. #zanata
msgid ""
msgstr ""
"Project-Id-Version: swift VERSION\n"
"Report-Msgid-Bugs-To: https://bugs.launchpad.net/openstack-i18n/\n"
"POT-Creation-Date: 2023-06-27 13:31+0000\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"PO-Revision-Date: 2016-07-18 10:20+0000\n"
"Last-Translator: Pablo Caruana <pcaruana@redhat.com>\n"
"Language: es\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"Generated-By: Babel 2.0\n"
"X-Generator: Zanata 4.3.3\n"
"Language-Team: Spanish\n"

#, python-format
msgid "Bad key for %(name)r: %(err)s"
msgstr "Clave errónea para  %(name)r: %(err)s"

#, python-format
msgid "Can not access the file %s."
msgstr "No se puede acceder al archivo %s."

#, python-format
msgid "Can not load profile data from %s."
msgstr "No se pueden cargar los datos de perfil desde %s."

#, python-format
msgid "Data download error: %s"
msgstr "Error de descarga de datos: %s"

msgid "Did not get a keys dict"
msgstr "No tuvimos un diccionario de claves"

msgid "ERROR get_keys() missing callback"
msgstr "ERROR get_keys() No se proporciona devolución de llamada "

msgid "Error listing devices"
msgstr "Error al mostrar los dispositivos"

#, python-format
msgid "Error on render profiling results: %s"
msgstr "Error al representar los resultados de perfil: %s"

msgid "Error parsing recon cache file"
msgstr "Error al analizar el archivo de memoria caché de recon"

msgid "Error reading recon cache file"
msgstr "Error al leer el archivo de memoria caché de recon"

msgid "Error reading ringfile"
msgstr "Error al leer el ringfile"

msgid "Error reading swift.conf"
msgstr "Error al leer swift.conf"

msgid "Error retrieving recon data"
msgstr "Error al recuperar los datos de recon"

msgid "Error: An error occurred"
msgstr "Error: se ha producido un error"

#, python-format
msgid "Following CNAME chain for  %(given_domain)s to %(found_domain)s"
msgstr "Siguiente cadena CNAME de %(given_domain)s a %(found_domain)s"

#, python-format
msgid "Mapped %(given_domain)s to %(found_domain)s"
msgstr "Se ha correlacionado %(given_domain)s con %(found_domain)s"

#, python-format
msgid "Missing key for %r"
msgstr "Falta una clave en %r"

#, python-format
msgid "Profiling Error: %s"
msgstr "Error de perfil: %s"

#, python-format
msgid "Ratelimit sleep log: %(sleep)s for %(account)s/%(container)s/%(object)s"
msgstr ""
"Ajuste de límite de registro de suspensión: %(sleep)s para %(account)s/"
"%(container)s/%(object)s"

#, python-format
msgid "Returning 497 because of blacklisting: %s"
msgstr "Se devuelven 497 debido a las listas negras: %s"

msgid "The file type are forbidden to access!"
msgstr "El acceso al tipo de archivo está prohibido."

msgid "Warning: Cannot ratelimit without a memcached client"
msgstr ""
"Aviso: no se puede ajustar el límite sin un cliente almacenado en memoria "
"caché"

#, python-format
msgid "method %s is not allowed."
msgstr "el método %s no está permitido."

msgid "no log file found"
msgstr "no se ha encontrado ningún archivo de registro"

msgid "odfpy not installed."
msgstr "odfpy no está instalado."

#, python-format
msgid "plotting results failed due to %s"
msgstr "error en el trazado de resultados debido a %s"

msgid "python-matplotlib not installed."
msgstr "python-matplotlib no está instalado."
